### PR TITLE
Package format v2 migration

### LINKED ISSUE #48: Dependency cleanup and package format migration
@ipa-mig 
As already done for all cob_bringup packages

### PR DESCRIPTION
**Pull request by mgruhler**
fixes #48 
also contains #49 

[Click Here for non-whitespace changes](https://github.com/ipa320/cob_navigation/pull/50/files?w=1)

### CODE REVIEW

File: cob_navigation_config/CMakeLists.txt [hunk 1]
```diff
@@ -5,19 +5,30 @@ find_package(catkin REQUIRED)
...
+  cob4-1
+  cob4-2
+  desire
+  raw3-1
+  raw3-2
+  raw3-3
+  raw3-4
+  raw3-5
+  raw3-6
+)
```
Comment (fmessmer) [inline cob_navigation_config/CMakeLists.txt:19]: we do not support some of the robots anymore...check cob_robots which of those are obsolete...

Comment (mgruhler) [inline cob_navigation_config/CMakeLists.txt:19]: So should I remove the unsupported ones?
I thought I'll just keep them in for now, as there are also several currently supported ones still missing...

**Comment by fmessmer** [comment]
Other than that: LGTM

File: cob_navigation_config/CMakeLists.txt (hunk 1 above)
Comment (mgruhler) [inline cob_navigation_config/CMakeLists.txt:19]: removed unsupported robots.
Robots as per `cob_robots` that are still missing are:
- cob3-9
- cob4-4
- cob4-6

**Comment by fmessmer** [comment]
alright...merging!

**Pull request merged**